docs/xml/elements/xml_data_best_practices.rst

```diff
@@ -0,0 +1,84 @@
+XML Data Best Practices
+=======================
+
+Following is a series of best practice for data collection and XML file creation and suggestions about best practices of
+formatting data within your VIP XML file.
+
+Element Identifiers
+-------------------
+
+Most elements within the VIP feed require unique identifiers, `xs:ID` data types, and in order to maintain uniqueness
```

Comment:
This is going to be a bit of a pain, but we should list off the prefixes we want states to use. The examples are great, but we need to be a bit exhaustive with this particular best practice.